inference_lib compile error

```
2020-04-21 00:12:12  [  6%] Built target data_feed_proto
2020-04-21 00:12:13  In file included from /workspace/Paddle/paddle/fluid/platform/place.h:21:0,
2020-04-21 00:12:13                   from /workspace/Paddle/paddle/fluid/platform/stream/cuda_stream.h:21,
2020-04-21 00:12:13                   from /workspace/Paddle/paddle/fluid/platform/stream/cuda_stream.cc:15:
2020-04-21 00:12:13  /workspace/Paddle/paddle/fluid/platform/variant.h:28:28: fatal error: boost/config.hpp: No such file or directory
2020-04-21 00:12:13   #include "boost/config.hpp"
2020-04-21 00:12:13                              ^
2020-04-21 00:12:13  compilation terminated.
2020-04-21 00:12:13  paddle/fluid/platform/stream/CMakeFiles/cuda_stream.dir/build.make:62: recipe for target 'paddle/fluid/platform/stream/CMakeFiles/cuda_stream.dir/cuda_stream.cc.o' failed
2020-04-21 00:12:13  make[3]: *** [paddle/fluid/platform/stream/CMakeFiles/cuda_stream.dir/cuda_stream.cc.o] Error 1
2020-04-21 00:12:13  CMakeFiles/Makefile2:5129: recipe for target 'paddle/fluid/platform/stream/CMakeFiles/cuda_stream.dir/all' failed
2020-04-21 00:12:13  make[2]: *** [paddle/fluid/platform/stream/CMakeFiles/cuda_stream.dir/all] Error 2
```
cuda_stream 缺少boost依赖
https://xly.bce.baidu.com/paddlepaddle/paddle/ipipe/#/detail/793245/job/0

--- a/paddle/fluid/platform/stream/CMakeLists.txt
+++ b/paddle/fluid/platform/stream/CMakeLists.txt
@@ -1,3 +1,3 @@
 IF(WITH_GPU)
-cc_library(cuda_stream SRCS cuda_stream.cc DEPS enforce)
+cc_library(cuda_stream SRCS cuda_stream.cc DEPS enforce boost)
 ENDIF()

